lib/controller/mixins/session-model.js

```diff
@@ -10,17 +10,19 @@ module.exports = Controller => class extends Controller {
     }
 
     createSessionModel(req, res, next) {
-        if (typeof req.session === 'undefined') {
-            throw new Error('Session is undefined');
+        if (!req.session) {
+            return next(new Error('Session is undefined'));
         }
 
         if (req.sessionModel) {
             req.sessionModel.destroy();
         }
 
-        req.sessionModel = new Model(null, {
+        req.sessionModel = new WizardModel(null, {
             session: req.session,
-            key: 'hmpo-wizard-' + req.form.options.name
+            key: 'hmpo-wizard-' + req.form.options.name,
+            journeyModel: req.journeyModel,
```

Comment:
WizardModel will throw an error, express will catch and pass to the errorHandler